specs/provider.yaml

```diff
@@ -913,6 +913,43 @@ paths:
           description: |
             If the request could not be processed successfully, a useful error
             message should be displayed to the user.
+
+  /resources/{id}/measures:
+    get:
+      summary: Get how much a resource has used its features
+      description: |
+        Manifold will call this endpoint daily to get usage information about a
+        resource and its features.
+
+        The Provider should only need to hold information about the time left
+        in the current month and the previous month.
+      parameters:
+        - $ref: '#/parameters/resource_id'
+        - name: period_start
+          in: query
+          required: true
+          type: string
+          format: datetime
+        - name: period_end
+          in: query
+          required: true
+          type: string
+          format: datetime
+      tags:
+        - Resource
+      responses:
+        200:
+          description: |
+            A response should contain a measures map in which the key represents a
+            resource feature and the value is the most updated usage value of the
```

Comment:
Add something like:
`This usage value is assumed to be the sum total of usage for that feature for the billing period`